packages/alea-frontend/pages/coverage-update.tsx

```diff
@@ -125,11 +125,11 @@ const CoverageUpdatePage: NextPage = () => {
               (e) => alert(e)
             );
           }}
-          sx={{ mt: '15px' }}
+          sx={{ mt: '15px',alignSelf:'flex-start',ml:'-170px'}}
```

Comment:
This '-170px' ml looks like a hack. Let's talk tomorrow to see the problem and fix it properly